fearless/Modules/WarningAlert/Configs/JsonLoadingFailedAlertConfig.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+extension WarningAlertConfig {
+    static func connectionProblemAlertConfig(with locale: Locale) -> WarningAlertConfig {
+        WarningAlertConfig(
+            title: R.string.localizable.connectionProblemsText(preferredLanguages: locale.rLanguages),
+            iconImage: R.image.iconWarning(),
```

Comment:
iconImage: R.image.iconWarning() - looks like default value, you provide this icon to every warning alert